### PR TITLE
Service Catalog Documentation

### PR DESCRIPTION
**Pull request by Kartikey-star**

### CODE REVIEW

File: docs/entities/servicecatalog.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# ServiceCatalog
+
+A Service Claim represents a group of Registered Services which are available in a Cluster Environment.
```
Comment (dperaza4dustbit) [inline docs/entities/servicecatalog.md:3]: Catalog not Claim

**Review by dperaza4dustbit — COMMENTED** [review]
Looks good, just one comment to fix

File: docs/entities/servicecatalog.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# ServiceCatalog
+
+A Service Catalog represents a group of Registered Services which are available in a Cluster Environment.
+
+## Specification
+
+The definition of a ServiceCatalog can be obtained directly from our [ServiceCatalog
+CRD](../../config/crd/bases/primaza.io_servicecatalogs.yaml). The specification
+contains a list of `ServiceCatalogService`. The Service Catalog Service contains the following fields:
```
Comment (filariow) [inline docs/entities/servicecatalog.md:9]: For a better review experience please format the markdown as follow. Oneline for each sentence, one empty line for creating a new paragraph.

```suggestion
The definition of a ServiceCatalog can be obtained directly from our [ServiceCatalog CRD](../../config/crd/bases/primaza.io_servicecatalogs.yaml).
The specification contains a list of `ServiceCatalogService`.
The Service Catalog Service contains the following fields:
```

**Review by filariow — CHANGES_REQUESTED** [review]
a few nits

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

**Pull request merged**